cmd/pitr/collector/collector.go

```diff
@@ -264,7 +266,10 @@ func (c *Collector) manageBinlog(binlog pxc.Binlog) (err error) {
 		return errors.Wrap(err, "put gtid-set object")
 	}
 
-	setBuffer.WriteString(set)
+	_, err = setBuffer.WriteString(set)
```

Comment:
here it's also always nil)

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/768/commits/548df709315de0137e70807e4884404a80524752